cinema/models.py

```diff
@@ -1,6 +1,10 @@
+import os.path
```

Comment:
It is common to import `os` and later call methods from it rather than importing `os.path`.